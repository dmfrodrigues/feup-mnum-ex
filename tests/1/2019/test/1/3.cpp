#include <bits/stdc++.h>
using namespace std;
double g3(double x){
  return log(x+11.0L);
}
int main(){
  double guess = -20.0L;
  cout << setprecision(10) << fixed;
  while(guess < 20.0L){
    double x = guess;
    for(int i = 0; i < 1000; ++i) x = g3(x);
    cout << "guess=" << guess << "\t, x=" << x << endl;
    guess += 0.1L;
  }
  return 0;
}
