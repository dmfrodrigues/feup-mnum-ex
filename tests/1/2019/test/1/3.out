guess=-20.0000000000	, x=nan
guess=-19.9000000000	, x=nan
guess=-19.8000000000	, x=nan
guess=-19.7000000000	, x=nan
guess=-19.6000000000	, x=nan
guess=-19.5000000000	, x=nan
guess=-19.4000000000	, x=nan
guess=-19.3000000000	, x=nan
guess=-19.2000000000	, x=nan
guess=-19.1000000000	, x=nan
guess=-19.0000000000	, x=nan
guess=-18.9000000000	, x=nan
guess=-18.8000000000	, x=nan
guess=-18.7000000000	, x=nan
guess=-18.6000000000	, x=nan
guess=-18.5000000000	, x=nan
guess=-18.4000000000	, x=nan
guess=-18.3000000000	, x=nan
guess=-18.2000000000	, x=nan
guess=-18.1000000000	, x=nan
guess=-18.0000000000	, x=nan
guess=-17.9000000000	, x=nan
guess=-17.8000000000	, x=nan
guess=-17.7000000000	, x=nan
guess=-17.6000000000	, x=nan
guess=-17.5000000000	, x=nan
guess=-17.4000000000	, x=nan
guess=-17.3000000000	, x=nan
guess=-17.2000000000	, x=nan
guess=-17.1000000000	, x=nan
guess=-17.0000000000	, x=nan
guess=-16.9000000000	, x=nan
guess=-16.8000000000	, x=nan
guess=-16.7000000000	, x=nan
guess=-16.6000000000	, x=nan
guess=-16.5000000000	, x=nan
guess=-16.4000000000	, x=nan
guess=-16.3000000000	, x=nan
guess=-16.2000000000	, x=nan
guess=-16.1000000000	, x=nan
guess=-16.0000000000	, x=nan
guess=-15.9000000000	, x=nan
guess=-15.8000000000	, x=nan
guess=-15.7000000000	, x=nan
guess=-15.6000000000	, x=nan
guess=-15.5000000000	, x=nan
guess=-15.4000000000	, x=nan
guess=-15.3000000000	, x=nan
guess=-15.2000000000	, x=nan
guess=-15.1000000000	, x=nan
guess=-15.0000000000	, x=nan
guess=-14.9000000000	, x=nan
guess=-14.8000000000	, x=nan
guess=-14.7000000000	, x=nan
guess=-14.6000000000	, x=nan
guess=-14.5000000000	, x=nan
guess=-14.4000000000	, x=nan
guess=-14.3000000000	, x=nan
guess=-14.2000000000	, x=nan
guess=-14.1000000000	, x=nan
guess=-14.0000000000	, x=nan
guess=-13.9000000000	, x=nan
guess=-13.8000000000	, x=nan
guess=-13.7000000000	, x=nan
guess=-13.6000000000	, x=nan
guess=-13.5000000000	, x=nan
guess=-13.4000000000	, x=nan
guess=-13.3000000000	, x=nan
guess=-13.2000000000	, x=nan
guess=-13.1000000000	, x=nan
guess=-13.0000000000	, x=nan
guess=-12.9000000000	, x=nan
guess=-12.8000000000	, x=nan
guess=-12.7000000000	, x=nan
guess=-12.6000000000	, x=nan
guess=-12.5000000000	, x=nan
guess=-12.4000000000	, x=nan
guess=-12.3000000000	, x=nan
guess=-12.2000000000	, x=nan
guess=-12.1000000000	, x=nan
guess=-12.0000000000	, x=nan
guess=-11.9000000000	, x=nan
guess=-11.8000000000	, x=nan
guess=-11.7000000000	, x=nan
guess=-11.6000000000	, x=nan
guess=-11.5000000000	, x=nan
guess=-11.4000000000	, x=nan
guess=-11.3000000000	, x=nan
guess=-11.2000000000	, x=nan
guess=-11.1000000000	, x=nan
guess=-11.0000000000	, x=nan
guess=-10.9000000000	, x=2.6108686381
guess=-10.8000000000	, x=2.6108686381
guess=-10.7000000000	, x=2.6108686381
guess=-10.6000000000	, x=2.6108686381
guess=-10.5000000000	, x=2.6108686381
guess=-10.4000000000	, x=2.6108686381
guess=-10.3000000000	, x=2.6108686381
guess=-10.2000000000	, x=2.6108686381
guess=-10.1000000000	, x=2.6108686381
guess=-10.0000000000	, x=2.6108686381
guess=-9.9000000000	, x=2.6108686381
guess=-9.8000000000	, x=2.6108686381
guess=-9.7000000000	, x=2.6108686381
guess=-9.6000000000	, x=2.6108686381
guess=-9.5000000000	, x=2.6108686381
guess=-9.4000000000	, x=2.6108686381
guess=-9.3000000000	, x=2.6108686381
guess=-9.2000000000	, x=2.6108686381
guess=-9.1000000000	, x=2.6108686381
guess=-9.0000000000	, x=2.6108686381
guess=-8.9000000000	, x=2.6108686381
guess=-8.8000000000	, x=2.6108686381
guess=-8.7000000000	, x=2.6108686381
guess=-8.6000000000	, x=2.6108686381
guess=-8.5000000000	, x=2.6108686381
guess=-8.4000000000	, x=2.6108686381
guess=-8.3000000000	, x=2.6108686381
guess=-8.2000000000	, x=2.6108686381
guess=-8.1000000000	, x=2.6108686381
guess=-8.0000000000	, x=2.6108686381
guess=-7.9000000000	, x=2.6108686381
guess=-7.8000000000	, x=2.6108686381
guess=-7.7000000000	, x=2.6108686381
guess=-7.6000000000	, x=2.6108686381
guess=-7.5000000000	, x=2.6108686381
guess=-7.4000000000	, x=2.6108686381
guess=-7.3000000000	, x=2.6108686381
guess=-7.2000000000	, x=2.6108686381
guess=-7.1000000000	, x=2.6108686381
guess=-7.0000000000	, x=2.6108686381
guess=-6.9000000000	, x=2.6108686381
guess=-6.8000000000	, x=2.6108686381
guess=-6.7000000000	, x=2.6108686381
guess=-6.6000000000	, x=2.6108686381
guess=-6.5000000000	, x=2.6108686381
guess=-6.4000000000	, x=2.6108686381
guess=-6.3000000000	, x=2.6108686381
guess=-6.2000000000	, x=2.6108686381
guess=-6.1000000000	, x=2.6108686381
guess=-6.0000000000	, x=2.6108686381
guess=-5.9000000000	, x=2.6108686381
guess=-5.8000000000	, x=2.6108686381
guess=-5.7000000000	, x=2.6108686381
guess=-5.6000000000	, x=2.6108686381
guess=-5.5000000000	, x=2.6108686381
guess=-5.4000000000	, x=2.6108686381
guess=-5.3000000000	, x=2.6108686381
guess=-5.2000000000	, x=2.6108686381
guess=-5.1000000000	, x=2.6108686381
guess=-5.0000000000	, x=2.6108686381
guess=-4.9000000000	, x=2.6108686381
guess=-4.8000000000	, x=2.6108686381
guess=-4.7000000000	, x=2.6108686381
guess=-4.6000000000	, x=2.6108686381
guess=-4.5000000000	, x=2.6108686381
guess=-4.4000000000	, x=2.6108686381
guess=-4.3000000000	, x=2.6108686381
guess=-4.2000000000	, x=2.6108686381
guess=-4.1000000000	, x=2.6108686381
guess=-4.0000000000	, x=2.6108686381
guess=-3.9000000000	, x=2.6108686381
guess=-3.8000000000	, x=2.6108686381
guess=-3.7000000000	, x=2.6108686381
guess=-3.6000000000	, x=2.6108686381
guess=-3.5000000000	, x=2.6108686381
guess=-3.4000000000	, x=2.6108686381
guess=-3.3000000000	, x=2.6108686381
guess=-3.2000000000	, x=2.6108686381
guess=-3.1000000000	, x=2.6108686381
guess=-3.0000000000	, x=2.6108686381
guess=-2.9000000000	, x=2.6108686381
guess=-2.8000000000	, x=2.6108686381
guess=-2.7000000000	, x=2.6108686381
guess=-2.6000000000	, x=2.6108686381
guess=-2.5000000000	, x=2.6108686381
guess=-2.4000000000	, x=2.6108686381
guess=-2.3000000000	, x=2.6108686381
guess=-2.2000000000	, x=2.6108686381
guess=-2.1000000000	, x=2.6108686381
guess=-2.0000000000	, x=2.6108686381
guess=-1.9000000000	, x=2.6108686381
guess=-1.8000000000	, x=2.6108686381
guess=-1.7000000000	, x=2.6108686381
guess=-1.6000000000	, x=2.6108686381
guess=-1.5000000000	, x=2.6108686381
guess=-1.4000000000	, x=2.6108686381
guess=-1.3000000000	, x=2.6108686381
guess=-1.2000000000	, x=2.6108686381
guess=-1.1000000000	, x=2.6108686381
guess=-1.0000000000	, x=2.6108686381
guess=-0.9000000000	, x=2.6108686381
guess=-0.8000000000	, x=2.6108686381
guess=-0.7000000000	, x=2.6108686381
guess=-0.6000000000	, x=2.6108686381
guess=-0.5000000000	, x=2.6108686381
guess=-0.4000000000	, x=2.6108686381
guess=-0.3000000000	, x=2.6108686381
guess=-0.2000000000	, x=2.6108686381
guess=-0.1000000000	, x=2.6108686381
guess=0.0000000000	, x=2.6108686381
guess=0.1000000000	, x=2.6108686381
guess=0.2000000000	, x=2.6108686381
guess=0.3000000000	, x=2.6108686381
guess=0.4000000000	, x=2.6108686381
guess=0.5000000000	, x=2.6108686381
guess=0.6000000000	, x=2.6108686381
guess=0.7000000000	, x=2.6108686381
guess=0.8000000000	, x=2.6108686381
guess=0.9000000000	, x=2.6108686381
guess=1.0000000000	, x=2.6108686381
guess=1.1000000000	, x=2.6108686381
guess=1.2000000000	, x=2.6108686381
guess=1.3000000000	, x=2.6108686381
guess=1.4000000000	, x=2.6108686381
guess=1.5000000000	, x=2.6108686381
guess=1.6000000000	, x=2.6108686381
guess=1.7000000000	, x=2.6108686381
guess=1.8000000000	, x=2.6108686381
guess=1.9000000000	, x=2.6108686381
guess=2.0000000000	, x=2.6108686381
guess=2.1000000000	, x=2.6108686381
guess=2.2000000000	, x=2.6108686381
guess=2.3000000000	, x=2.6108686381
guess=2.4000000000	, x=2.6108686381
guess=2.5000000000	, x=2.6108686381
guess=2.6000000000	, x=2.6108686381
guess=2.7000000000	, x=2.6108686381
guess=2.8000000000	, x=2.6108686381
guess=2.9000000000	, x=2.6108686381
guess=3.0000000000	, x=2.6108686381
guess=3.1000000000	, x=2.6108686381
guess=3.2000000000	, x=2.6108686381
guess=3.3000000000	, x=2.6108686381
guess=3.4000000000	, x=2.6108686381
guess=3.5000000000	, x=2.6108686381
guess=3.6000000000	, x=2.6108686381
guess=3.7000000000	, x=2.6108686381
guess=3.8000000000	, x=2.6108686381
guess=3.9000000000	, x=2.6108686381
guess=4.0000000000	, x=2.6108686381
guess=4.1000000000	, x=2.6108686381
guess=4.2000000000	, x=2.6108686381
guess=4.3000000000	, x=2.6108686381
guess=4.4000000000	, x=2.6108686381
guess=4.5000000000	, x=2.6108686381
guess=4.6000000000	, x=2.6108686381
guess=4.7000000000	, x=2.6108686381
guess=4.8000000000	, x=2.6108686381
guess=4.9000000000	, x=2.6108686381
guess=5.0000000000	, x=2.6108686381
guess=5.1000000000	, x=2.6108686381
guess=5.2000000000	, x=2.6108686381
guess=5.3000000000	, x=2.6108686381
guess=5.4000000000	, x=2.6108686381
guess=5.5000000000	, x=2.6108686381
guess=5.6000000000	, x=2.6108686381
guess=5.7000000000	, x=2.6108686381
guess=5.8000000000	, x=2.6108686381
guess=5.9000000000	, x=2.6108686381
guess=6.0000000000	, x=2.6108686381
guess=6.1000000000	, x=2.6108686381
guess=6.2000000000	, x=2.6108686381
guess=6.3000000000	, x=2.6108686381
guess=6.4000000000	, x=2.6108686381
guess=6.5000000000	, x=2.6108686381
guess=6.6000000000	, x=2.6108686381
guess=6.7000000000	, x=2.6108686381
guess=6.8000000000	, x=2.6108686381
guess=6.9000000000	, x=2.6108686381
guess=7.0000000000	, x=2.6108686381
guess=7.1000000000	, x=2.6108686381
guess=7.2000000000	, x=2.6108686381
guess=7.3000000000	, x=2.6108686381
guess=7.4000000000	, x=2.6108686381
guess=7.5000000000	, x=2.6108686381
guess=7.6000000000	, x=2.6108686381
guess=7.7000000000	, x=2.6108686381
guess=7.8000000000	, x=2.6108686381
guess=7.9000000000	, x=2.6108686381
guess=8.0000000000	, x=2.6108686381
guess=8.1000000000	, x=2.6108686381
guess=8.2000000000	, x=2.6108686381
guess=8.3000000000	, x=2.6108686381
guess=8.4000000000	, x=2.6108686381
guess=8.5000000000	, x=2.6108686381
guess=8.6000000000	, x=2.6108686381
guess=8.7000000000	, x=2.6108686381
guess=8.8000000000	, x=2.6108686381
guess=8.9000000000	, x=2.6108686381
guess=9.0000000000	, x=2.6108686381
guess=9.1000000000	, x=2.6108686381
guess=9.2000000000	, x=2.6108686381
guess=9.3000000000	, x=2.6108686381
guess=9.4000000000	, x=2.6108686381
guess=9.5000000000	, x=2.6108686381
guess=9.6000000000	, x=2.6108686381
guess=9.7000000000	, x=2.6108686381
guess=9.8000000000	, x=2.6108686381
guess=9.9000000000	, x=2.6108686381
guess=10.0000000000	, x=2.6108686381
guess=10.1000000000	, x=2.6108686381
guess=10.2000000000	, x=2.6108686381
guess=10.3000000000	, x=2.6108686381
guess=10.4000000000	, x=2.6108686381
guess=10.5000000000	, x=2.6108686381
guess=10.6000000000	, x=2.6108686381
guess=10.7000000000	, x=2.6108686381
guess=10.8000000000	, x=2.6108686381
guess=10.9000000000	, x=2.6108686381
guess=11.0000000000	, x=2.6108686381
guess=11.1000000000	, x=2.6108686381
guess=11.2000000000	, x=2.6108686381
guess=11.3000000000	, x=2.6108686381
guess=11.4000000000	, x=2.6108686381
guess=11.5000000000	, x=2.6108686381
guess=11.6000000000	, x=2.6108686381
guess=11.7000000000	, x=2.6108686381
guess=11.8000000000	, x=2.6108686381
guess=11.9000000000	, x=2.6108686381
guess=12.0000000000	, x=2.6108686381
guess=12.1000000000	, x=2.6108686381
guess=12.2000000000	, x=2.6108686381
guess=12.3000000000	, x=2.6108686381
guess=12.4000000000	, x=2.6108686381
guess=12.5000000000	, x=2.6108686381
guess=12.6000000000	, x=2.6108686381
guess=12.7000000000	, x=2.6108686381
guess=12.8000000000	, x=2.6108686381
guess=12.9000000000	, x=2.6108686381
guess=13.0000000000	, x=2.6108686381
guess=13.1000000000	, x=2.6108686381
guess=13.2000000000	, x=2.6108686381
guess=13.3000000000	, x=2.6108686381
guess=13.4000000000	, x=2.6108686381
guess=13.5000000000	, x=2.6108686381
guess=13.6000000000	, x=2.6108686381
guess=13.7000000000	, x=2.6108686381
guess=13.8000000000	, x=2.6108686381
guess=13.9000000000	, x=2.6108686381
guess=14.0000000000	, x=2.6108686381
guess=14.1000000000	, x=2.6108686381
guess=14.2000000000	, x=2.6108686381
guess=14.3000000000	, x=2.6108686381
guess=14.4000000000	, x=2.6108686381
guess=14.5000000000	, x=2.6108686381
guess=14.6000000000	, x=2.6108686381
guess=14.7000000000	, x=2.6108686381
guess=14.8000000000	, x=2.6108686381
guess=14.9000000000	, x=2.6108686381
guess=15.0000000000	, x=2.6108686381
guess=15.1000000000	, x=2.6108686381
guess=15.2000000000	, x=2.6108686381
guess=15.3000000000	, x=2.6108686381
guess=15.4000000000	, x=2.6108686381
guess=15.5000000000	, x=2.6108686381
guess=15.6000000000	, x=2.6108686381
guess=15.7000000000	, x=2.6108686381
guess=15.8000000000	, x=2.6108686381
guess=15.9000000000	, x=2.6108686381
guess=16.0000000000	, x=2.6108686381
guess=16.1000000000	, x=2.6108686381
guess=16.2000000000	, x=2.6108686381
guess=16.3000000000	, x=2.6108686381
guess=16.4000000000	, x=2.6108686381
guess=16.5000000000	, x=2.6108686381
guess=16.6000000000	, x=2.6108686381
guess=16.7000000000	, x=2.6108686381
guess=16.8000000000	, x=2.6108686381
guess=16.9000000000	, x=2.6108686381
guess=17.0000000000	, x=2.6108686381
guess=17.1000000000	, x=2.6108686381
guess=17.2000000000	, x=2.6108686381
guess=17.3000000000	, x=2.6108686381
guess=17.4000000000	, x=2.6108686381
guess=17.5000000000	, x=2.6108686381
guess=17.6000000000	, x=2.6108686381
guess=17.7000000000	, x=2.6108686381
guess=17.8000000000	, x=2.6108686381
guess=17.9000000000	, x=2.6108686381
guess=18.0000000000	, x=2.6108686381
guess=18.1000000000	, x=2.6108686381
guess=18.2000000000	, x=2.6108686381
guess=18.3000000000	, x=2.6108686381
guess=18.4000000000	, x=2.6108686381
guess=18.5000000000	, x=2.6108686381
guess=18.6000000000	, x=2.6108686381
guess=18.7000000000	, x=2.6108686381
guess=18.8000000000	, x=2.6108686381
guess=18.9000000000	, x=2.6108686381
guess=19.0000000000	, x=2.6108686381
guess=19.1000000000	, x=2.6108686381
guess=19.2000000000	, x=2.6108686381
guess=19.3000000000	, x=2.6108686381
guess=19.4000000000	, x=2.6108686381
guess=19.5000000000	, x=2.6108686381
guess=19.6000000000	, x=2.6108686381
guess=19.7000000000	, x=2.6108686381
guess=19.8000000000	, x=2.6108686381
guess=19.9000000000	, x=2.6108686381
